### ISSUE TITLE
Links: These need to have underlines. Just blue is not accessible. Also need inline option that is same color as text.

### ISSUE THREAD

**Issue by mjabbink** [carbon-design-system/gatsby-theme-carbon#150]

**Comment by alisonjoseph** [comment]
We are just using the link component directly from Carbon https://www.carbondesignsystem.com/components/link/code

@dakahn any input on the a11y for this?

**Comment by vpicone** [comment]
Moving this to the Carbon repo because we use their Link component and it should be resolved there.

I believe this is an actual issue for inline links. Per WCAG:

>Link underlines or some other non-color visual distinction are required (when the links are discernible to those with color vision).

> While some links may be visually evident from page design and context, such as navigational links, links within text are often visually understood only from their own display attributes. Removing the underline and leaving only the color difference for such links would be a failure because there would be no other visual indication (besides color) that it is a link.

https://www.w3.org/TR/WCAG20-TECHS/F73.html
